docs/source/ko/package_reference/hf_api.md

```diff
@@ -0,0 +1,126 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# HfApi Client[[hfapi-client]]
+
+아래는 허깅 페이스 허브의 API를 위한 파이썬 래퍼인 `HfApi` 클래스에 대한 문서입니다.
+
+`HfApi`의 모든 메서드는 패키지의 루트에서 직접 접근할 수 있습니다. 두 접근 방식은 아래에서 자세히 설명합니다.
+
+루트 메서드를 사용하는 것이 더 간단하지만 [`HfApi`] 클래스를 사용하면 더 유연하게 사용할 수 있습니다.
+특히 모든 HTTP 호출에서 재사용할 토큰을 전달할 수 있습니다. 
+이 방식은 토큰이 머신에 유지되지 않기 때문에 `huggingface-cli login` 또는 [`login`]를 사용하는 방식과는 다르며,
+다른 엔드포인트를 제공하거나 사용자정의 에이전트를 구성할 수도 있습니다.
+
+```python
+from huggingface_hub import HfApi, list_models
+
+# Use root method
```

Comment:
I think it'd be a better user experience for Korean readers if the code comments here are also translated. What do you think?

Reply:
Hi @stevhliu 👋
I agree with your idea. 
When I looked at the documentation in other languages, the code comments were not translated, so we left them untranslated, but I think it would be more helpful for Koreans to read the documentation if the code comments are translated.